### PR TITLE
[vmware-monitoring] collect additional hostsystem metrics

### PR DESCRIPTION
**Pull request by ninePebbles**
Add CPU metrics:
* cpu_coreutilization_percentage
* cpu_corepeak_usage

Add network metrics:
* network_bytes_rx_kbps
* network_bytes_tx_kbps

Bump chart

### CODE REVIEW

**Comment by ninePebbles** [comment]
@viennaa Unfortunately I don't have permissions to suggest reviewers, leaving a note here.

File: system/vmware-monitoring/templates/collector-configmap.yaml [hunk 1]
```diff
@@ -176,6 +176,10 @@ data:
         key: "cpu|capacity_provisioned"
       - metric_suffix: "cpu_corecount_provisioned"
         key: "cpu|corecount_provisioned"
+      - metric_suffix: "cpu_corepeak_usage"
+        key: "cpu|peak_cpu_core_usage"      
+      - metric_suffix: "cpu_coreutilization_percentage"
+        key: "cpu|coreUtilization_average"
```
Comment (viennaa) [inline system/vmware-monitoring/templates/collector-configmap.yaml:182]: Make sure you add the unit at the end, if possible. I suppose in these examples it is percentage, no?

```suggestion
      - metric_suffix: "cpu_peak_core_usage_percentage"
        key: "cpu|peak_cpu_core_usage"      
      - metric_suffix: "cpu_core_utilization_percentage"
        key: "cpu|coreUtilization_average"
```

**Review by viennaa — COMMENTED** [review]
Thanks for providing this PR. I would try to stay as close as possible to the origin key and add the unit in the end. Suggestions attached, think about what fits best.

File: system/vmware-monitoring/templates/collector-configmap.yaml (hunk 1 above)
Comment (ninePebbles) [reply system/vmware-monitoring/templates/collector-configmap.yaml:182]: I wasn't sure about the unit in  'cpu|peak_cpu_core_usage', VROPS documentation is not clear about that. These look good to me, thanks.

Comment (viennaa) [reply system/vmware-monitoring/templates/collector-configmap.yaml]: i haven't looked it up, so you better double check ;)

Comment (viennaa) [reply system/vmware-monitoring/templates/collector-configmap.yaml]: Can spot it either, even when I am looking into an active vROps deployment. Question is, what do you expect from it, when you don't know, what it is about?

Comment (ninePebbles) [reply system/vmware-monitoring/templates/collector-configmap.yaml]: Hopefully a percentage, if it's MHz instead, then we can compute the percentage ourselves...

**Comment by viennaa** [comment]
@ninePebbles can you maintain your name in github please?

**Pull request merged**